src/phoenix/datasets/dataset.py

```diff
@@ -121,6 +124,142 @@ def from_name(cls, name: str) -> "Dataset":
         schema = Schema.from_json(schema_json)
         return cls(df, schema, name)
 
+    @classmethod
+    def from_open_inference(cls, dataframe: DataFrame) -> "Dataset":
+        schema = Schema()
+        column_renaming: Dict[str, str] = {}
+        for group_name, group in groupby(
+            sorted(
+                map(_parse_open_inference_column_name, dataframe.columns),
+                key=lambda column: column.name,
+            ),
+            key=lambda column: column.name,
+        ):
+            open_inference_columns = list(group)
+            if group_name == "":
+                column_names_by_category = {
+                    column.category: column.full_name for column in open_inference_columns
+                }
+                schema = replace(
+                    schema,
+                    prediction_id_column_name=column_names_by_category.get("id"),
+                    timestamp_column_name=column_names_by_category.get("timestamp"),
+                )
+                continue
```

Comment:
In the situation where the name is null, do we want to use the full name? That would result in column names like `:id.id:`. Maybe we can fallback to the specifier if it exists, and then the category as a last resort.

Reply:
this is fine for now because these column won't be shown in the UI